adafruit_portalbase/network.py

```diff
@@ -392,12 +398,12 @@ def get_io_data(self, feed_key):
         io_client = self._get_io_client()
 
         while True:
-            try:
-                return io_client.receive_all_data(feed_key)
-            except RuntimeError as exception:
-                print("An error occured, retrying! 1 -", exception)
-                continue
-            break
+            # try:
```

Comment:
I'm not sure if I understand fully why the try/except is no longer needed here, I think it moved to elsewhere maybe?

It does seem to work okay though, we can get rid of the commented out try/except maybe?

Reply:
Oh yeah, I didn't mean to leave that commented out.